internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -122,6 +122,7 @@ func (r *AgentServiceConfigReconciler) Reconcile(ctx context.Context, req ctrl.R
 		r.ensureAgentLocalAuthSecret,
 		r.ensurePostgresSecret,
 		r.ensureIngressCertCM,
+		r.ensureAssistedCM,
```

Comment:
Up around line 85:

```diff
+ // +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
```

and then rebuild the manifests with `export ENABLE_KUBE_API && make generate-manifests`. I assume that we already have the necessary permissions but we should use the operator-sdk markers.